ale/drivers/mgs_drivers.py

```diff
@@ -12,6 +12,140 @@
 from ale import util
 
 
+class MgsMocNarrowAngleCameraIsisLabelNaifSpiceDriver(LineScanner, IsisLabel, NaifSpice, RadialDistortion, Driver):
+    """
+    Driver for reading MGS MOC WA ISIS labels.
+    """
+
+    @property
+    def instrument_id(self):
+        """
+        Returns an instrument id for uniquely identifying the instrument, but often
+        also used to be piped into Spice Kernels to acquire IKIDs. Therefore they
+        the same ID the Spice expects in bods2c calls.
+        Expects instrument_id to be defined in the IsisLabel mixin. This should be
+        a string of the form 'MOC-NA'
+
+        Returns
+        -------
+        : str
+          instrument id
+        """
+        id_lookup = {
+        "MOC-NA" : "MGS_MOC_NA"
+        }
+        return id_lookup[super().instrument_id]
+
+
+    @property
+    def sensor_name(self):
+        """
+        Returns
+        -------
+        : String
+          The name of the sensor
+        """
+        return self.instrument_id
+
+    @property
+    def ephemeris_start_time(self):
+        """
+        Returns the ephemeris start time of the image.
+        Expects spacecraft_id to be defined. This should be the integer
+        Naif ID code for the spacecraft.
+
+        Returns
+        -------
+        : float
+          ephemeris start time of the image
+        """
+        if not hasattr(self, '_ephemeris_start_time'):
+            sclock = self.label['IsisCube']['Instrument']['SpacecraftClockCount']
+            self._ephemeris_start_time = spice.scs2e(self.spacecraft_id, sclock)
+        return self._ephemeris_start_time
+
+    @property
+    def ephemeris_stop_time(self):
+        """
+        ISIS doesn't preserve the spacecraft stop count that we can use to get
+        the ephemeris stop time of the image, so compute the ephemeris stop time
+        from the start time and the exposure duration.
+        """
+        return self.ephemeris_start_time + (self.exposure_duration/1000 * ((self.image_lines) * self.label['IsisCube']['Instrument']['DowntrackSumming']))
+
+    @property
+    def detector_start_sample(self):
+        """
+        Returns
+        -------
+        : int
+          The starting detector sample of the image
+        """
+        return self.label['IsisCube']['Instrument']['FirstLineSample']
+
+
+    @property
+    def detector_center_sample(self):
+        """
+        Returns the center detector sample. Expects ikid to be defined. This should
+        be an integer containing the Naif Id code of the instrument.
+
+        Returns
+        -------
+        : float
+          Detector sample of the principal point
+        """
+        return float(spice.gdpool('INS{}_CENTER'.format(self.ikid), 0, 1)[0])
```

Comment:
Should probably change this to getting 2 elements instead of 1 to be consistent with detector_center_line

Reply:
I'm only using element [0] in detector_center_sample, so shouldn't I just get one element, the first one?
In detector_center_line I get two elements because I need to get both in order to access the second one.